ocrd_zip.md

```diff
@@ -127,25 +130,26 @@ be referenced in a `mets:file/mets:Flocat` in the `mets.xml`.
 To pack a workspace to OCRD-ZIP:
 
 * Create a temporary folder `TMP`
-* Copy mets.xml to `TMP/data/mets.xml`
-* Foreach `mets:file` `f` in `TMP/data/mets.xml`:
-  * If it is not a `file://`-URL
-    * If `Ocrd-Manifestation-Depth` is `partial`
+* Foreach `mets:file` `f` in the source METS:
+  * If it is not a `file://`-URL and not a file path
+    * If `Ocrd-Manifestation-Depth` is `partial`,
       continue
   * Download/Copy the file to a location within `TMP/data`. The structure SHOULD be `<USE>/<ID>` where
     * `<USE>` is the `USE` attribute of the parent `mets:fileGrp`
     * `<ID>` is the `ID` attribute of the `mets:file`
-  * Replace the URL of `f` with `file:///data/<USE>/<ID>` in
-    * all `mets:FLocat` of `TMP/data/mets.xml`
+  * Replace the URL of `f` with the path relative to `/data` (SHOULD be `<USE>/<ID>`) in
+    * all `mets:FLocat` of the METS
     * all other files in the workspace, esp. PAGE-XML
+* Write out the changed METS to `TMP/data/mets.xml` (or another location in `TMP/data/`, specified by [`Ocrd-Mets`](#ocrd-mets)
```

Comment:
Is the bagit container now defined as workspace?
The 'source METS file' is the only METS file that can be contained in the Bagit container, as all other METS files that may be present are not referenced in the 'source METS file'.
If so there is no need to rename it.

Reply:
> Is the bagit container now defined as workspace?

`/data` should be also useable as a workspace.

> If so there is no need to rename it.

In real life, the METS filename will usually be meaningful, like an ID of the work or such. It seems sensible to offer a way to preserve that.

But we could handle that transparently and consistenly name that file `data/mets.xml` within the OCRD-ZIP and only rename it on spilling the bag. Would that work for you?

Reply:
> It seems sensible to offer a way to preserve that.

I think so, too.
If there is only one METS file inside the bagit container how do we avoid duplicates in file names.
If there are multiple METS files inside a workspace/directory, which one is the 'right' one? This information gets lost when the context (bagit container or workflow) gets lost.